ajax/libs/kube/package.json

```diff
@@ -16,10 +16,11 @@
   "autoupdate": {
     "source": "git",
     "target": "git://github.com/imperavi/kube.git",
-    "basePath": "",
+    "basePath": "dist",
     "files": [
       "css/*",
-      "js/*"
+      "js/*",
+      "scss/*"
```

Comment:
I think we don't need filed under `scss` folder. Let's use `"!(scss)/*"` instead of above.